### ISSUE TITLE
InfluxDB 2.6

### ISSUE THREAD

**Issue by jeffreyssmith2nd** [influxdata/docs-v2#4665]
ETA: December 15 2022

### Bug Fixes

1. [46464f4](https://github.com/influxdata/influxdb/commit/46464f4): Optimize SHOW FIELD KEY CARDINALITY
1. [666cabb](https://github.com/influxdata/influxdb/commit/666cabb): Fix wrong max age transformation from seconds
1. [f026d7b](https://github.com/influxdata/influxdb/commit/f026d7b): Fixes migrating when a remote already exists
1. [9bf8840](https://github.com/influxdata/influxdb/commit/9bf8840): Update me and users routes to match cloud/documentation
1. [ade21ad](https://github.com/influxdata/influxdb/commit/ade21ad): Restrict file permissions by default
1. [ffd069a](https://github.com/influxdata/influxdb/commit/ffd069a): Handle NaN in scraper

### Features

1. [7708108](https://github.com/influxdata/influxdb/commit/7708108): Port check-schema and merge-schema from 1.x
1. [c2eac86](https://github.com/influxdata/influxdb/commit/c2eac86): Port report-db command from 1.x
1. [853d615](https://github.com/influxdata/influxdb/commit/853d615): Perform basic package validation

### Other

1. [e62c8ab](https://github.com/influxdata/influxdb/commit/e62c8ab): Chore: upgrade to Go 1.18.8
1. [86207fe](https://github.com/influxdata/influxdb/commit/86207fe): Build(flux): update flux to v0.189.0
1. [61870e5](https://github.com/influxdata/influxdb/commit/61870e5): Chore: update 2.5 changelog
1. [4de89af](https://github.com/influxdata/influxdb/commit/4de89af): Refactor: remove dead iterator code
1. [07e6ef2](https://github.com/influxdata/influxdb/commit/07e6ef2): Build(flux): update flux to v0.191.0
1. [7ad8fba](https://github.com/influxdata/influxdb/commit/7ad8fba): Chore: fix trace message text
1. [ef098ac](https://github.com/influxdata/influxdb/commit/ef098ac): Chore: cleanup codeowners file
1. [26daa86](https://github.com/influxdata/influxdb/commit/26daa86): Chore: bump testcontainers to latest released version
1. [cee487f](https://github.com/influxdata/influxdb/commit/cee487f): Chore: update Go to 1.18.9

**Comment by jeffreyssmith2nd** [comment]
I do expect one or two more changes to go in before release, but this is the large majority of what is planned.

**Comment by lwandzura** [comment]
@jeffreyssmith2nd  for the influxd inspect report-db utility is it exactly like this one:
https://docs.influxdata.com/influxdb/v1.8/tools/influx_inspect/#report

we'll need to create a page for that in 2.6 so I need all the flags/options

**Comment by jeffreyssmith2nd** [comment]
Not quite, it's closer to `report-db` which is only in the Enterprise builds before now. Here's the help text with all the flag explanations.

```
./bin/darwin/influxd inspect report-db --help
Estimates cloud 2 cardinality for a database

Usage:
  influxd inspect report-db [flags]

Flags:
      --c int            set worker concurrency, defaults to one (default 1)
      --db-path string   path to database
      --detailed         include counts for fields, tags
      --exact            report exact counts
  -h, --help             help for report-db
      --rollup string    rollup level - t: total, b: bucket, r: retention policy, m: measurement (default "m")
```

**Comment by lwandzura** [comment]
is check-schema also an influxd inspect command? if so can you provide the usage and flags for that as well?

**Comment by jeffreyssmith2nd** [comment]
Yes, it is influx inspect as well
```
./bin/darwin/influxd inspect check-schema --help
Check for conflicts in the types between shards

Usage:
  influxd inspect check-schema [flags]

Flags:
      --conflicts-file string   Filename conflicts data should be written to (default "conflicts.json")
  -h, --help                    help for check-schema
      --log-level Log-Level     The level of logging used througout the command (default info)
      --path string             Path under which fields.idx files are located (default ".")
      --schema-file string      Filename schema data should be written to (default "schema.json")
```
And merge-schema
```
./bin/darwin/influxd inspect merge-schema --help

Merge a set of schema files from the check-schema command

Usage:
  influxd inspect merge-schema [flags]

Flags:
      --conflicts-file string   Filename conflicts data should be written to (default "conflicts.json")
  -h, --help                    help for merge-schema
      --schema-file string      Filename for the output file (default "schema.json")
```
